backend/python/app/services/implementations/story_service.py

```diff
@@ -51,3 +52,37 @@ def create_story(self, story, content):
         db.session.refresh(new_story)
 
         return new_story
+
+    def get_story_translations(self, user_id, translate):
+        if translate:
+            return (
+                db.session.query(
+                    Story.id.label("story_id"),
+                    Story.title.label("title"),
+                    Story.description.label("description"),
+                    Story.level.label("level"),
+                    StoryTranslation.id.label("story_translation_id"),
+                    StoryTranslation.language.label("language"),
+                    StoryTranslation.stage.label("stage"),
+                    StoryTranslation.translator_id.label("translator_id"),
+                    StoryTranslation.reviewer_id.label("reviewer_id"),
+                )
+                .join(StoryTranslation, Story.id == StoryTranslation.story_id)
+                .filter(StoryTranslation.translator_id == user_id)
+            )
+        else:
+            return (
+                db.session.query(
+                    Story.id.label("story_id"),
+                    Story.title.label("title"),
+                    Story.description.label("description"),
+                    Story.level.label("level"),
+                    StoryTranslation.id.label("story_translation_id"),
+                    StoryTranslation.language.label("language"),
+                    StoryTranslation.stage.label("stage"),
+                    StoryTranslation.translator_id.label("translator_id"),
+                    StoryTranslation.reviewer_id.label("reviewer_id"),
+                )
+                .join(StoryTranslation, Story.id == StoryTranslation.story_id)
+                .filter(StoryTranslation.reviewer_id == user_id)
```

Comment:
You could throw in a ternary operator into the filter to avoid repeat code, eg
`.filter(StoryTranslation.translator_id== user_id if translate else StoryTranslation.reviewer_id == user_id)`